apps/dialtone-documentation/docs/components/card.md

```diff
@@ -22,9 +22,9 @@ storybook: https://dialtone.dialpad.com/vue/?path=/story/components-card--defaul
       <div>Content slot. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec fermentum molestie semper. Morbi finibus nulla turpis, nec molestie mi rutrum.</div>
     </div>
     <div class="d-card__footer">
-      <div class="d-d-flex d-g8">
+      <dt-stack direction="row" gap="400">
```

Comment:
I don't expect you to review every instance that was migrated, but you can skim through them to pick up on the pattern the migration script does. 

I've gone through all doc pages and confirmed them.